src/content/content.npmjs.js

```diff
@@ -4,18 +4,20 @@ import { mountContentScript, reloadWhenURLChanged } from './content';
 import { fetchPackageInfo } from './content-events';
 import { urlParsers } from './registry/npm';
 
+const CSS_SELECTOR_INSTALL = '.c84e15be';
```

Comment:
I don't feel comfortable with this selector. It seems to be an auto-generated one, and as such, it might be replaced with another one in the next deploy of npmjs.com.

Can you find another selector?